plugins/tutor-contrib-library-authoring-mfe/tutor_library_authoring_mfe/plugin.py

```diff
@@ -0,0 +1,98 @@
+from glob import glob
+import os
+import pkg_resources
+import uuid
+
+from tutor import hooks as tutor_hooks
+
+from .__about__ import __version__
+
+########################################
+# CONFIGURATION
+########################################
+
+tutor_hooks.Filters.CONFIG_DEFAULTS.add_items(
+    [
+        ("LIBRARY_AUTHORING_MICROFRONTEND_VERSION", __version__),
+        ("LIBRARY_AUTHORING_MFE_APP", {
+            "name": "library-authoring",
+            "repository": "https://github.com/openedx/frontend-app-library-authoring",
+            "port": 3001,
+        })
+    ]
+)
```

Comment:
As of Palm there's a new way to do this:

```suggestion
port = 3001

@MFE_APPS.add()
def _add_my_mfe(mfes):
    mfes["library-authoring"] = {
        "repository": "https://github.com/openedx/frontend-app-library-authoring",
        "port": port,
    }
    return mfes

hooks.Filters.ENV_TEMPLATE_VARIABLES.add_items(
    [("library_authoring_port", port)]
)
```